### PR TITLE
Datarows windows fix

### PR DESCRIPTION
**Pull request by zjelveh**

### CODE REVIEW

File: namematch/predict.py [hunk 1]
```diff
@@ -98,14 +98,16 @@ def get_potential_edges(self, dr_file, match_models, model_info,
             params (Parameters obj):  contains parameter values (i.e. use_uncovered_phats)
         '''
         try:
-            thread = dr_file.split('/')[-1].replace('data_rows_', '').replace('.parquet', '')
+            import pathlib
+            p = pathlib.Path(dr_file)
+            thread = p.parts[-1].replace('data_rows_', '').replace('.parquet', '')
+            #thread = dr_file.split('/').replace('data_rows_', '').replace('.parquet', '')
```
Comment (tweddielin) [inline namematch/predict.py:104]: We can just remove the commented out line.

**Review by tweddielin — COMMENTED** [review]
@zjelveh thanks for the fix! Things look good and I'll wait for @mmcneill to test on windows machine. Only minor suggestion for the PEP8 convention that a comment should start with a # and a single space. 

I think we want to write some guidelines in contributing.md about using some linter (eg. flake8) or formatter to enforce PEP8 style.

File: namematch/predict.py (hunk 1 above)
Comment (zjelveh) [reply namematch/predict.py]: done

**Comment by tweddielin** [comment]
@zjelveh Can you add a note in the README saying that to use python 3.8 for windows? Because nmslib currently doesn't work with python 3.9.

**Review by mmcneill — APPROVED** [review]
Thanks Zubin -- I was able to run this branch on my windows computer after getting past the Visual C++ requirement (using python version < 3.9 so that nmslib could install).

**Review by tweddielin — APPROVED** [review]

**Comment by mmcneill** [comment]
> Can you add a note in the README saying that to use python 3.8 for windows? Because nmslib currently doesn't work with python 3.9.

I can take care of this

**Pull request merged**